### ISSUE TITLE
RFC: Completely redo monospace metrics, reduce to 80% current width

### ISSUE THREAD

**Issue by alerque** [alerque/libertinus#375]
One of the things I noticed (and that initially [made me think it wasn't even working](https://github.com/alerque/libertinus/issues/214#issuecomment-706506751)) when testing the CTAN packages for Libertinus Fonts such as [libertinus-otf](https://ctan.org/pkg/libertinus-otf) is that in addition to setting up fake slant and bold variants for Mono, it also stretches the entire font (including the default regular weight roman original) by a factor of 0.8.

The result IMHO looks *far* better than the original. The extreme width is always something that's bothered me about the mono. Truth be told I rarely use it at all, and its metric indiscretions are one reason for that.

I propose that this change to 80% of the current width (or thereabouts) be done in the source and become the new default width for a future (major version) release of the font. Alternatively I suppose to so much breakage an automatic process could transform the current Mono into a Mono Condensed font to add to future releases.

cc: @hvoss49

## Before / After Comparison

![image](https://user-images.githubusercontent.com/173595/95649848-71a10300-0ae8-11eb-8fda-2e4acbb1b676.png)

**Comment by mbaz** [comment]
I agree the current spacing is too large. I seldom use Libertinus Mono because of this. But maybe 80% is too large?

I think a mono font should "look" the same size as the other fonts. While the "before" mono font seems larger compared to Serif, the "after" seems smaller to me.

How do 85% and 90% scaling look?

**Comment by hvoss49** [comment]
For the Monofont I always use Scale=MatchLowercase (of the regular serif font) and FakeStretch=0.8. For the Libertinus it makes also sense to use Scale=MatchUppercase. However, the original dimensions of the monofont for width and height doesn't really lokk nice.

**Comment by shawnbrown** [comment]
I quite like the width of Libertinus Mono as it is--I use it for programming every day. I've looked at the 85% and 80% scaling and it's easy for busy code to look cramped. Maybe there could be a "narrow" variant? Or maybe scale Libertinus Mono but keep the original-width version as "Libertinus Code"?

**Comment by iandoug** [comment]
My 2c :-)

Would be interested in a Sans mono... find serif monos too busy for programming. Maybe I need to increase font size or may be age/eye related. YMMV.

If you are going to improve mono, the ascii tilde could be more distinct than a slightly wavy dash ... Hack is distinct but may be a bit radical tor these styles.
https://github.com/source-foundry/Hack

Cheers, Ian

**Comment by shawnbrown** [comment]
@iandoug, I can see that. I gravitated to Libertinus Mono because of the serifs. Monospace fonts with serifs tend to use slab serifs--which I particularly want to avoid. Originally I was coding with Century Schoolbook Mono but it's a little busy but moved to Linux Libertine because it's not as cramped. And then I discovered the improved Libertinus Mono ([sample screenshot](https://imgur.com/a/NOYwnpZ)).
